### ISSUE TITLE
Language-dependent issue: parenthesis tokenization in English

### ISSUE THREAD

**Issue by LanceNorskog** [explosion/spaCy#3453]
## Feature description
How do you expect spaCy to tokenize this sentence?
What do you expect the parse tree to look like?
> There are three things in air, Nitrogen (79%), oxygen (20%), and other types of gases (1%).

I suppose my question is: do you believe that the current spaCy can tokenize, apply POS, and create a parse tree correctly for this sentence?

(This sentence is from the Simplified Wikipedia.)

**Comment by ines** [comment]
I just tested you example with the latest version, and the tokenization looks like this:

```python
['There', 'are', 'three', 'things', 'in', 'air', ',', 'Nitrogen', '(', '79', '%', ')', ',', 
'oxygen', '(', '20', '%', ')', ',', 'and', 'other', 'types', 'of', 'gases', '(', '1', '%', ')', '.']
```

Here's the parse tree, which looks decent to me. (If you want to try this out yourself, note that I set `options={'collapse_punct': False, 'compact': True}` to show all punctuation tokens separately and make the big tree easier to read.)

![displacy](https://user-images.githubusercontent.com/13643239/54751833-66237980-4bdc-11e9-9e61-c27620249390.png)

That said, you can probably also find a lot of edge cases and examples of where the out-of-the-box tokenization rules can't generalise for all combinations of punctuation and parentheses, or where the pre-trained parser or tagger makes an incorrect prediction. So if you're dealing with longer inserts in parentheses and the parser struggles with those, you might want to fine-tune it with more examples like that.

Looking at a single sentence in isolation isn't very helpful, because it doesn't give you a good idea of the overall accuracy on your data and what to focus on. Even if you train a fancy state-of-the-art model that gets 90% accuracy on your data, it still means that every 10th prediction it makes is wrong.

**Comment by LanceNorskog** [comment]
Thanks, @ines ! Also, please look at https://github.com/explosion/spaCy/issues/3454 in which someone else mentions a similar oddity with square brackets. On stackoverflow someone gave a hack to collapse (35%) into one token, and I can see that this sort of thing does warrant special per-use-case attention. Another example is 3:2 (ratio).

An aside: is percent really a noun? Are all units nouns? (I'm not an NLP expert.)
